README.md

```diff
@@ -1,5 +1,4 @@
-# Introduction to FP in Scala
-
+# Introduction to FP in Scala [![Build Status](https://travis-ci.org/tmcgilchrist/introduction-to-fp-in-scala.svg?branch=travis_ci)](https://travis-ci.org/tmcgilchrist/introduction-to-fp-in-scala)
```

Comment:
Can you change these urls to ambiata and the branch to master, then looks good. Thanks.